### ISSUE TITLE
[Feature Request] Bypass online store password

### ISSUE THREAD

**Issue by cka29** [Shopify/checkout-sheet-kit-swift#151]
Hello,

I did not where to put this except here. According to https://github.com/Shopify/checkout-sheet-kit-swift/blob/249b80f7ecfa29b632cb8dd85b8eee284dd7165d/.github/CONTRIBUTING.md#proposing-features it's possible to propose features :) 

We are heavy users of the new native checkout sheet and already using this in production. Is there any way to bypass the online stores password protection? 

Our use case is simple:
- Online Store is closed (password protected) due to mobile app-only pre-sale
- Native Checkout should work (while store is still password protected)

Thanks!

**Comment by kaisegni** [comment]
I'm running into a similar issue,
We have dev store password which is password protected, and entering the password redirects to the main shop url instead of continue to checkout, any way to by pass that?

**Comment by kiftio** [comment]
Just to confirm, this is still planned

**Comment by cka29** [comment]
@kiftio any news regarding this? Thank you 🙏

**Comment by EricNetsch** [comment]
Yes we need this as well - causing a lot of issues with merchants who use PW protection before big sales
